module-db/queries/notifications/QueryNotificationsIncrement.hpp

```diff
@@ -7,19 +7,22 @@
 #include <Common/Query.hpp>
 #include <string>
 #include <PhoneNumber.hpp>
+#include <cstdint>
 
 namespace db::query::notifications
 {
     class Increment : public Query
     {
         const NotificationsRecord::Key key;
         const utils::PhoneNumber::View number;
+        const ssize_t count;
 
       public:
-        Increment(NotificationsRecord::Key key, const utils::PhoneNumber::View &number);
+        Increment(NotificationsRecord::Key key, const utils::PhoneNumber::View &number, const ssize_t count = 1);
 
         [[nodiscard]] auto getKey() const noexcept -> NotificationsRecord::Key;
         [[nodiscard]] auto getNumber() const noexcept -> const utils::PhoneNumber::View &;
+        [[nodiscard]] auto getCount() const noexcept -> ssize_t;
```

Comment:
now we can have a negative count?

Reply:
Well, it's a first attempt at abstracting the decrement operation. Ideally, we'd have a facade for incrementing and decrementing particular notifications.

Reply:
We can pass a negative count.

Reply:
Negative values are no longer needed. Reworked - added special query to handle decrementing of notification count.